### ISSUE TITLE
GLSL Compilation Errors at incorrect lines

### ISSUE THREAD

**Issue by OxygenL602** [Pen-Group/penPlus-shader-editor#14]
Console very often reports errors on blank lines (screenshot as follows)
![image](https://github.com/user-attachments/assets/ad3e3314-2477-45e8-8165-d9d63eb708d4)

**Comment by OxygenL602** [comment]
and also, This is a fresh compilation and I did not alter the code

**Comment by AlexSchoolOH** [comment]
I know what is happening. I don't really know how to fix it. It happens because WEBGL's compiler takes lines that are blank or commented out and skips them completely. If I can figure out a fix I will.

**Comment by LegoBrainBiker** [comment]
> I know what is happening. I don't really know how to fix it. It happens because WEBGL's compiler takes lines that are blank or commented out and skips them completely. If I can figure out a fix I will.

I don't think that is the issue. I think it is because of how you format it to sepperate it into a vertex and fragment shader the lines get mixed up. blank lines DO count.
